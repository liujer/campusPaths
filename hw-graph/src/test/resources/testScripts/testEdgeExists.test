# Tests method edgeExists by adding nodes and edges
# to a graph and checking if particular edges exist
# between the nodes
CreateGraph A
AddNode A one
AddNode A two
AddEdge A one two e12
EdgeExists A one two
EdgeExists A two one