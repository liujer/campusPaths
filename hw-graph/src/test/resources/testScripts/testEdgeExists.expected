# Tests method edgeExists by adding nodes and edges
# to a graph and checking if particular edges exist
# between the nodes
Created graph A
Added node one to A
Added node two to A
Added edge e12 to A
Edge from one to two exists
Edge from two to one does not exist